eng/CredScanSuppression.json

```diff
@@ -40,9 +40,10 @@
                 "eng/common/testproxy/dotnet-devcert.pfx",
                 "sdk/confidentialledger/azure-confidentialledger/tests/_shared/constants.py",
                 "sdk/keyvault/azure-keyvault-certificates/tests/ca.key",
-                "sdk/identity/azure-identity/tests/ec-certificate.pem",
                 "sdk/identity/azure-identity/tests/certificate.pfx",
                 "sdk/identity/azure-identity/tests/certificate-with-password.pfx",
+                "sdk/identity/azure-identity/tests/ec-certificate.pem",
+                "sdk/identity/azure-identity/tests/helpers.py",
```

Comment:
Can we put these into a file that is only intended for these fake secrets and is named as such. Right now by blindly adding this helpers.py file to the suppression list we risk another potential real leak in that file because it is used for other things beyond fake secrets.

Reply:
Sure, I can split this into a `fake_credentials` (or similar) file, that seems reasonable.